plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -494,3 +628,14 @@ private fun ModuleSourceInfo.toRemoteArtifact(): RemoteArtifact? =
             null
         }
     }
+
+private fun AnalyzerConfiguration.getConanFactory() =
+    determineEnabledPackageManagers()
+        .filter { it.descriptor.id.startsWith("Conan") }
+        .let { managers ->
+            require(managers.size < 2) {
+                "All of the $managers managers are able to manage 'Conan' projects. Please enable only one of them."
```

Comment:
Why is this check needed? Can't we just say: Bazel only recognizes Conan dependencies if also Conan is enabled?

Reply:
The same logic also exist in Pub, I did it for alignment.

Reply:
In Pub it only exists because there are `Gradle` and `GradleInspector` and not both of them can be enabeld at the same time.